src/renderer/App.tsx

```diff
@@ -214,7 +220,7 @@ class _App extends React.Component<connectedProps, typeof AppDefaults.state> {
         return <GraphContainer />;
       case "listview":
         return <ListView />;
-      case "synthesisOutlineEditor":
+      case "synthesisOutlineEditor" && featureToggles.showDocList:
```

Comment:
instead of && featureToggles.showDocList put an if/else in the case block.
case 'editor':
if
else
break